Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 gem 'jruby-openssl', platforms: :jruby
-gem 'rake'
+gem 'rake', '~> 10'
```

Comment:
This was reverted in 11.1.0 so this is no longer necessary https://github.com/ruby/rake/blob/master/History.rdoc#1110--2016-03-11